src/Microsoft.AspNet.Mvc.Core/FileStreamResult.cs

```diff
@@ -13,48 +13,48 @@ namespace Microsoft.AspNet.Mvc
 {
     /// <summary>
     /// Represents an <see cref="ActionResult"/> that when executed will
-    /// write a file from a stream to the response.
+    /// write a stream to the response.
     /// </summary>
-    public class FileStreamResult : FileResult
+    public class StreamResult : FileResult
```

Comment:
Instead should we have `StreamResult` and also `FileStreamResult` which would derive from `StreamResult`?
@rynowak